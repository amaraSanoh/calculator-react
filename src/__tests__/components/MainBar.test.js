import React from 'react';
import renderer from 'react-test-renderer';
import MainBar from '../../components/MainBar';

describe('<MainBar /> Component Tests', () => {
    it('has 1 child', () => {
      const tree = renderer.create(<MainBar />).toJSON();
      expect(tree.children.length).toBe(1);
    });

    it('has makeStyles-root-1 as a className', () => {
        const tree = renderer.create(<MainBar />).toJSON();
        expect(tree.props.className).toBe('makeStyles-root-1');
    });

    it('renders correctly', () => {
        const tree = renderer.create(<MainBar />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});