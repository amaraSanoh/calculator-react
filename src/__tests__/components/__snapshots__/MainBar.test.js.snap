// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainBar /> Component Tests renders correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <h6
        className="MuiTypography-root makeStyles-title-3 MuiTypography-h6"
      >
        Calculator
      </h6>
    </div>
  </header>
</div>
`;
